routes/welcome.js

```diff
@@ -0,0 +1,46 @@
+const express = require("express");
+const router = express.Router();
+
+router.get("/", (req, res) => {
+  res.render("index");
```

Comment:
Note, even though this isn't in `routes/clucks.js`, you can still `res.render("clucks/index");`